contents/teams/content/objectives.mdx

```diff
@@ -1,76 +1,125 @@
 ## Editorial
 
 <details>
-  <summary>Awesome content and docs for data warehouse</summary>
+  <summary>2x-ing product comparisons</summary>
 
-**Owner:** <TeamMember name="Ian Vanagas" /> (support from <TeamMember name="Vincent Ge" /> and <TeamMember name="Edwin Lim" />)
+**Owner:** Natalia Amorim
 
-**Motivation:** Data warehouse is a high-leverage product for our customer infrastructure strategy, but it lacks both high-quality docs and strong content marketing content to drive adoption. 
+**Motivation:** We want to make product comparisons consistent, up to date, and aligned with our positioning as a developer tool, so they become a reliable entry point for SEO and LLM visibility.
 
 **What we'll ship:**
-* Updated and fully fleshed out docs, working along similar lines to reference docs Edwin and Vincent are creating for error tracking.  
-* Alternatives, comparison and top-of-funnel content promoting our data warehouse.  
-* Opinionated blog and newsletter content (e.g. how we use data warehouse)
+
+* Audit and refresh most popular comparisons to ensure accuracy, consistency, developer-first positioning, LLM readability, etc.  
+* Ship missing error tracking comparisons against key competitors
```

Comment:
Note from our side:
- Would be great if comparison tables in this content was made reusable somehow. We'd love a version of this in certain places in docs. I think overview is as top funnel as real market pages and this would be a great addition to answer common questions

Reply:
I think @corywatilo is also working on something like this, so if you do, best to coordinate with him.

Reply:
Each product now has a [features array](https://github.com/PostHog/posthog.com/blob/6b6eae68e9d5b8665b082d7809edfe645db4215f/src/hooks/productData/error_tracking.tsx#L232) in json which lists out each competitor and has a value for each of the features.

At some point I want to get the comparison blog posts to reference this data so everything is maintained in the same place.

There are some other things worth considering:

1. How do we limit the features or grouping of features if you want to limit to a subset of features?
2. Does it make more sense to have a single file for each competitor (instead of being split into each of our products) so that we can do more comprehensive competitor comparisons that are by _company_ and not by _product_?